src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Form/Entity/ChooseEntityTypeType.php

```diff
@@ -27,19 +27,25 @@
 
 class ChooseEntityTypeType extends AbstractType
 {
+    private $formCount = 0;
+
     /**
      * @SuppressWarnings(PHPMD.UnusedLocalVariable) $key, $index are never used in the choice_attr callback
      * @param FormBuilderInterface $builder
      * @param array $options
      */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
+        $this->formCount++;
+
+        /** @var ChooseEntityTypeCommand $command */
+        $command = $builder->getData();
+        $choices = $command->getProtocolChoices();
```

Comment:
Couldn't the options come from the ProtocolChoiceFactory itself?

Reply:
See: https://github.com/SURFnet/sp-dashboard/pull/275#discussion_r304302958